README.md
# js_faq
 FAQ - Frequently Asked Questions 


ChangeLog
2016-11-23  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Added composer file
	* Released version 5.4.6

2016-11-05  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Extension Manual Update
	* Include dependent js_paginate extension for pagination.
	* https://typo3.org/extensions/repository/view/js_paginate
	* Released version 5.4.5

2016-10-05  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Add Manual.
	* Released version 5.4.4

2016-09-29  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Minor changes.
	* Released version 5.4.3

2016-08-13  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Minor changes.
	* Released version 5.4.2

2016-07-09  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Include dependent js_paginate extension for pagination.
	* Change viewhelper "{namespace pg=JS\JsFaq\ViewHelpers}" to "{namespace pg=JS\JsPaginate\ViewHelpers}"
	* https://typo3.org/extensions/repository/view/js_paginate
	* Released version 5.4.1

2016-06-25  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Change ordering based upon backend selection of images (media) and related FAQ.
	* Change error notification messages.
	* Change typoscript path to typo3conf/ext/js_faq/Configuration/TypoScript/Overrides/
	* Modify CSS.
	* Other Minor changes.
	* Released version 5.4.0

2016-06-24  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Added language files for German.
	* Create separate file for additional information.
	* Released version 5.3.0

2016-06-17  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Minor chanages and small bug fixes.
	* Released version 5.2.4

2016-05-31  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Added option for enable or disable Show All / Hide all button.
	* Released version 5.2.3

2016-05-20  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Pagination Enable / Disable functionality will manage via plugin option.
	* Added option for display number of record per page.
	* Released version 5.2.2

2016-05-18  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Manage additional headerdata via typoscript.
	* Released version 5.2.1

2016-05-05  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Added recursive functionality for starting point . 
	* Will display data from parent folder as well as child folders.
	* Minor changes.
	* Released version 5.2.0

2016-04-06  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Modify javascript for toggle. 
	* More than one plugin will work smoothly in same page.
	* Released version 5.1.1

2016-03-27  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Added plugin information.
	* Released version 5.1.0

2016-03-23  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Compatibility with TYPO3 8.0 
	* Released version 5.0.0

2016-04-06  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Compatibility with TYPO3 7.6 
	* Released version 4.0.0

2016-02-26  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Major Changes.
	* Image dimension manage via plugin and typoscript.
	* Bug fixes.
	* Changes in css.
	* After upgrade plugin you must need to update plugin configuration.
	* Released version 3.0.0

2016-02-08  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Major Changes.
	* FAL image support.
	* After upgrade plugin you must need to update all images in "Answer" area.
	* Released version 2.0.0

2016-01-01  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Added functionality to show FAQ between specific start-date and end-date.
	* Released version 1.6.4

2015-12-19  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Added elements to the Content Element Wizard. 
	* Released version 1.6.3

2015-07-21  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Added "Display FAQ - Category group wise" option in plugin.
	* Display FAQ - group of category wise in frontend module.
	* Released version 1.6.2

2015-07-16  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Added categories Tab options in plugin.
	* Added "Categories" option in plugin.
	* Display FAQ list depends upon category selected in plugin option.
	* Released version 1.6.1

2015-07-03  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Create new table for Category and set relation with FAQ table.
	* Added category option in FAQ and choose category from category selection box in Detail tab.
	* Added "Display Category" option in plugin.
	* Display category name in frontend module.
	* Released version 1.6.0

2015-06-02  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Added ordering by "Asked by" and "Expert" option in plugin option.
	* Order by : Asked by and Expert.
	* Released version 1.5.4

2015-05-28  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Added display setting for detail page of FAQ
	* Added option in plugin for "Display additional Information in detail page".
	* Added "Display Expert", "Display Asked by", "Display Related Link", "Display Related FAQ" option is checked in plugin then only display in frontend module.
	* Released version 1.5.3

2015-05-25  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Added "Display additional Information in detail page".
	* Released version 1.5.2

2015-05-18  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Add detail link on related FAQ option in front end module.
	* Display details of FAQ when click on related FAQ link.
	* Released version 1.5.1

2015-05-11  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Added related FAQ options in FAQ table.
	* Choose, Add and Edit related FAQ in FAQ table.
	* Added "Display Related FAQ" option in plugin options.
	* "Display Related FAQ" option is checked in plugin then only display in frontend module.
	* Released version 1.5.0

2015-05-04  Jainish Senjaliya, <jainishsenjaliya@gmail.com>

	* Added related link field in FAQ table.

ext_emconf.php
<?php

/***************************************************************
 * Extension Manager/Repository config file for ext: "js_faq"
 *
 * Auto generated by Extension Builder 2016-05-31
 *
 * Manual updates:
 * Only the data in the array - anything else is removed by next write.
 * "version" and "dependencies" must not be touched!
 ***************************************************************/

$EM_CONF[$_EXTKEY] = array(
	'title' => 'FAQ - Frequently Asked Questions',
	'description' => ' FAQ - Frequently Asked Questions',
	'category' => 'fe',
	'author' => 'Jainish Senjaliya',
	'author_email' => 'jainishsenjaliya@gmail.com',
	'state' => 'stable',
	'internal' => '',
	'uploadfolder' => '1',
	'createDirs' => '',
	'clearCacheOnLoad' => 0,
	'version' => '5.4.6',
	'constraints' => array(
		'depends' => array(
			'typo3' => '6.2.0 - 8.9.99',
			'js_paginate' => '1.0.3 - 3.9.99',
		),
		'conflicts' => array(
		),
		'suggests' => array(
		),
	),
);